src/main/java/network/brightspots/rcv/ContestConfig.java

```diff
@@ -518,6 +517,12 @@ private void validateCvrFileSources() {
             isValid = false;
             Logger.log(Level.SEVERE, "CDF files must be tabulated individually.");
           }
+          if (!source.getFilePath().endsWith(".xml") && !source.getFilePath().endsWith(".json")) {
+            Logger.log(Level.SEVERE,
+                "Unexpected file extension: %s.  CDF source files must be .xml or .json",
+                source.getFilePath());
```

Comment:
Nit (to avoid long filepaths obscuring the important part of the message):
```
Logger.severe("CDF source files must be .json or .xml! Unexpected file extension for: %s", source.getFilePath());
```